repoman/updates.py

```diff
@@ -77,6 +77,23 @@ def __init__(self, parent):
         updates_grid.attach(self.checks_grid, 0, 2, 1, 1)
         self.checks_grid.show()
 
+        automatic_title = Gtk.Label.new(_('Automatic Updates'))
+        automatic_title.set_margin_top(12)
+        automatic_title.set_halign(Gtk.Align.START)
+        Gtk.StyleContext.add_class(automatic_title.get_style_context(), 'h2')
+        updates_grid.attach(automatic_title, 0, 3, 1, 1)
+
+        automatic_label = Gtk.Label.new(_('Enable or configure automatic updates in system settings'))
```

Comment:
I also spotted that this sentence is missing a period, if we go the repoman route.